app/main.py

```diff
@@ -1,3 +1,40 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __add__(self, other):
+        return Distance(other + self.km)
```

Comment:
It works only if `other` is an integer. If it is a `Distance`, the method tries to add a `Distance` object and an integer inside the constructor. 
As a result, you create a `Distance` object passing another Distance to the constructor.

Could you use an `if` construction to avoid such a problem?